conda.recipe/build.sh

```diff
@@ -0,0 +1,3 @@
+SRC_DIR=$RECIPE_DIR/..
+cd $SRC_DIR
```

Comment:
you are missing npm-related steps here... check: https://github.com/damianavila/RISE#development (the same goes for the `bld.bat` file.